api/keepalive/index.ts

```diff
@@ -0,0 +1,28 @@
+import axios from 'axios'
+import * as express from 'express'
+import {app} from '../application'
+import { getConfigValue } from '../configuration'
+import { COOKIE_TOKEN } from '../configuration/references'
+
+const cookieToken = getConfigValue(COOKIE_TOKEN)
+
+async function handleAddressRoute(req, res) {
+  try {
+    const tokenSet = await app.locals.client.refresh(req.session.passport.user.tokenset.refresh_token)
```

Comment:
Not sure on the implementation, but you may want to feature toggle this route too (just in case)

Reply:
This is already feature toggled @paddybasi